intellijJVersions.gradle

```diff
@@ -64,10 +64,46 @@ static def ideProfiles() {
                     ]
                 ],
                 "RD": [
-                    sdkVersion: "RD-2020.1-SNAPSHOT",
+                    sdkVersion  : "RD-2020.1-SNAPSHOT",
                     rdGenVersion: "0.201.69",
                     nugetVersion: "2020.1.0-eap05",
-                    plugins: [
+                    plugins     : [
+                        "org.jetbrains.plugins.yaml"
+                    ]
+                ]
+            ]
+        ],
+        "2020.2": [
+            "sinceVersion": "202",
+            "untilVersion": "202.*",
+            "products"    : [
+                "IC": [
+                    sdkVersion: "IC-202.6250.13-EAP-SNAPSHOT",
+                    plugins   : [
+                        "org.jetbrains.plugins.terminal",
+                        "org.jetbrains.plugins.yaml",
+                        "PythonCore:202.6250.13",
+                        "java",
+                        "com.intellij.gradle",
+                        "org.jetbrains.idea.maven",
+                        "Docker:202.6250.6"
+                    ]
+                ],
+                "IU": [
+                    sdkVersion: "IU-202.6250.13-EAP-SNAPSHOT",
+                    plugins   : [
+                        "org.jetbrains.plugins.terminal",
+                        "Pythonid:202.6250.21",
+                        "org.jetbrains.plugins.yaml",
+                        "JavaScript",
+                        "JavaScriptDebugger",
```

Comment:
Does `com.intellij.database` work?

Reply:
Opps, ya, branch predates its adding to 201 since I just did a reabse I forgot about it.